### DISCUSSION TITLE
Bale collection and merged fields

### DISCUSSION THREAD

**Discussion by realm174** [Courseplay/courseplay#6882] [Q&A] [answered]
First of all, the bale collection works fabulously. Got it going on a large field, and had no issues, other than the occasional driving over a bale.

I have 4 small fields (1 to 4) that have been merged into one. I basically set it up as any other fields and I picked 1 for the field number on the first page. It actually picked up about 1/3 of all the bales, even the ones outside of the "1" field. But then it stopped.

How should I set it up to work on merged fields, or is this not an option yet?  My work around at the moment is that I setup 4 individual courses and run each one on each individual fields.

**Comment by Tensuko** [comment, marked as answer]
Did you create a User generated Field (Scan Field border and give it a number) and try it that way ?

**Reply by realm174** [reply]
I must be blind, I can't see the "Scan field border" option.  I did a "Start course recording" and drove around the entire area, which generated a temp course. Then, I saved it and gave it a name. But I can't see any option to scan or give it a field number.

**Reply by realm174** [reply]
Disregard.. I found the option and that appears to have fixed my issue. thanks!!
